dev/807cb705-5a00-47fa-b2a3-63aca7415820/device.lp

```diff
@@ -0,0 +1,21 @@
+(librepcb_device 807cb705-5a00-47fa-b2a3-63aca7415820
+ (name "LF442-MILACN")
+ (description "LF442-MIL Dual Low Power JFET Input Operational Amplifier With PDIP Footprint")
+ (keywords "")
```

Comment:
Maybe you could add a few keywords? For example `lf442,pdip,jfet,amplifier`. Maybe you can also think of others.